src/ui/EpisodesList/EpisodesList.tsx

```diff
@@ -21,6 +24,26 @@ export function EpisodesList({ className = '', episodes }: EpisodesListProps) {
 
   return (
     <div className={`${styles.container} ${className}`} ref={containerRef}>
+      {children}
+      <FeatureToggle feature="episodesFilter">
+        <div className={styles.episodeListOptions}>
```

Comment:
The episodes filter also gave me an idea to fix the virtual list by putting the content by rendering the episode info inside the list item component as the scroll seems to have been fixed.